src/styles/filters-history.scss

```diff
@@ -133,24 +130,18 @@
 	}
 	&__share-icon {
 		position: relative;
-		& i {
-			position: absolute;
-			top: 50%;
-			left: 50%;
-			transform: translate(-50%, -50%);
-			opacity: 0.5;
-			@include icon(url(../../resources/icons/share-option.svg), 14px, 14px);
-		}
+		width: 14px;
```

Comment:
when inspecting the share-icon element, it shows a giant box 500x500. not sure how to fix it, and whether it is important.